InterfaceF/src/Server/ClientHandler.java

```diff
@@ -0,0 +1,95 @@
+package Server;
+
+import java.io.DataInputStream;
+import java.io.DataOutputStream;
+import java.io.IOException;
+import java.net.Socket;
+import java.sql.SQLException;
+
+public class ClientHandler {
+    //к каждому клиенту сокета создается свой поток ввывода и ввода данных ин и оут
+    private Server server;
+    private Socket socket;
+
+    private DataInputStream in;
+    private DataOutputStream out;
+
+    public ClientHandler(Server server, Socket socket) {
+        try {
+            this.server = server;
+            this.socket = socket;
+
+            this.in = new DataInputStream(socket.getInputStream());
+            this.out = new DataOutputStream(socket.getOutputStream()); //формируется входящий и исходящий поток данных
+            //для ввода DataOutput это способ чтоб работала кодировка более правильный чем в эхосервере
+            new Thread(() -> {
+                boolean flagAuto=false;
+                try {
+                    String lNikName="_";
+                    while (true) {
+
+                            String str = in.readUTF();
+                        if(flagAuto==true) {
+                           String[] newNikname=str.split(" ");
+                            System.out.println("Сообщение от клиента_" + lNikName + ": " + str);
+                            if (str.equals("/end")) {
+                                break;
+                            }
+                            if ( newNikname[0].equals("/newNik")){
+                                DataBass.findAndCheang(lNikName,newNikname[1]);
+                                lNikName=newNikname[1];
+                            }
+                            // String idS=String.valueOf(id);
+                            //  str=idS+str;
+                            server.broadcastMsg(str, lNikName); //рассылка сообщений и для этого в констрктор мы передаем обьект сервера
+                        }else{
+
+
+                            String[] autors=str.split(" ");
+                            boolean exFlag=true;
+                             lNikName=DataBass.Search(autors[0], autors[1]);
```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [Nm: Method names should start with a lower case letter (NM_METHOD_NAMING_CONVENTION)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#nm-method-names-should-start-with-a-lower-case-letter-nm-method-naming-convention).
Methods should be verbs, in mixed case with the first letter lowercase, with the first letter of each internal word capitalized.